src/actions/AddSiteActions.js

```diff
@@ -85,6 +96,20 @@ export const updatePriceOption = ({priceOption}) => {
     }
 };
 
+export const updateCellProviderOption = ({cellProviderOption}) => {
+    return {
+        type: SITE_CELL_PROVIDER_CHANGED,
+        payload: {cellProviderOption}
+    }
+};
+
+export const updateCellStrengthOption = ({cellStrengthOption}) => {
+    return {
+        type: SITE_CELL_STRENGTH_CHANGED,
+        payload: {cellStrengthOption}
+    }
```

Comment:
All of these above follow the same pattern we've already had in place for text and picker updates.